etl/schema.py

```diff
@@ -1,21 +1,18 @@
-from pyspark.sql.types import StructType, StringType
+from pyspark.sql.types import StructType, StringType, ArrayType, MapType, LongType
 
 
-class CsvDataset:
+class ParquetDataset:
     schema = (
         StructType()
-        .add("pandas_idx", StringType(), True)
```

Comment:
Nice that we got rid of this!

FYI: these changes to spark & hql code will need to be incorporated into https://gitlab.wikimedia.org/gmodena/platform-airflow-dags/-/tree/multi-project-dags-repo/image-matching

Reply:
I'm actually not going to merge this PR into this repo but keep this one consistent with version one of the algorithm. I'll make PR on your GitLab repo